imsv-docs-docusaurus/openapi/endpoints/cards/models/card.yaml

```diff
@@ -20,45 +46,38 @@ properties:
       - cancelled
       - created
       - shipped
-  cardType:
+  fundingSourceIds:
+    type: array
+    description: List of Funding Source IDs linked to the card
+  expiry:
     type: string
-    description: The type of the card
-    enum:
-      - virtual
+    description: Expiry date of the card of physical cards
+    example: "202409"
+  currency:
+    type: string
+    description: The currency of this card
+    example: "USDC | USD"
   panFirst6:
     type: string
-    description: The first 6 digits of the card
+    description: First 6 digits of the card Primary Account Number
```

Comment:
This is a bit pedantic, but would this read better as 'First 6 digits of the card's PAN (Primary Account Number)?